GTFO_VR/Injections/GameHooks/InjectWaterTankShaderRemoval.cs

```diff
@@ -0,0 +1,42 @@
+﻿using HarmonyLib;
+using UnityEngine;
+
```

Comment:
I think if you gate it behind a config option that defaults to turning this off it should be fine for now.